base-template.sh

```diff
@@ -378,29 +392,35 @@ process_shared_hooks() {
 #####################################################
 update_shared_hooks_if_appropriate() {
     # run an init/update if we are after a "git pull" or triggered manually
-    if [ "$HOOK_NAME" = "post-merge" ] || [ "$HOOK_NAME" = ".githooks.shared.trigger" ]; then
+    GIT_NULL_REF="0000000000000000000000000000000000000000"
+
+    RUN_UPDATE="false"
+    [ "$HOOK_NAME" = "post-merge" ] && RUN_UPDATE="true"
+    [ "$HOOK_NAME" = ".githooks.shared.trigger" ] && RUN_UPDATE="true"
+    [ "$HOOK_NAME" = "post-checkout" ] && [ "$1" = "$GIT_NULL_REF" ] && RUN_UPDATE="true"
+    echo "Hook name: $HOOK_NAME, $1"
```

Comment:
Is this a debugging statement maybe that was left in?

Reply:
Ou jeah, damm!

Reply:
Thx for beeing, harsh :-)

Reply:
Hehe, no, thanks for fixing it up!
I'll try to get it merged in an hour or so. 🤞